app/apps/server/bridges/activation.js

```diff
@@ -20,13 +20,10 @@ export class AppActivationBridge {
 	}
 
 	async appStatusChanged(app, status) {
-		const { _id, username } = Users.findOne({ appId: app.getID() });
+		const { _id, username } = Users.findOneByAppId(app.getID());
```

Comment:
sry, I missed this tip before 🙈 

but since you're using only `_id` and `username` fields, you can request only them from DB, this will make the query faster and a lot less data will go from DB to RC (thankfully you've already added an options param):
```suggestion
		const { _id, username } = Users.findOneByAppId(app.getID(), { fields: { username: 1 } });
```
you doesn't need to request the `_id` field as Mongo always returns it 😉

Reply:
Good to know it, thanks! 👍